src/main/java/seedu/notus/data/timetable/Event.java

```diff
@@ -167,6 +170,26 @@ public boolean getRecurring() {
         return isRecurring;
     }
 
+    public ArrayList<String> getReminderPeriodsString() {
+        ArrayList<String> periods = new ArrayList<>();
+
+        ArrayList<Integer> dayRepeatPeriod = reminderPeriods.get(REMINDER_DAY);
+        ArrayList<Integer> weekRepeatPeriod = reminderPeriods.get(REMINDER_WEEK);
```

Comment:
Actually, what would happen if reminderPeriods that is stored was null? Should have a checker here if not a empty arraylist will be given

Reply:
Yup Ive added a check for it